README.md

```diff
@@ -57,7 +69,7 @@ a) Set up the job as follows:
 cd (build_path)/GEOSldas/install/bin
```

Comment:
Is this correct?

Reply:
I suppose this would need to be something like cd (build_path)/GEOSldas/install-SLES12/bin
Let's fix it when we include the SLES15 vs SLES12 instructions in #693